cfme/tests/candu/test_utilization_metrics.py

```diff
@@ -438,3 +461,35 @@ def test_utilization_breadcrumbs(appliance):
     view = navigate_to(appliance.collections.utilization, "All")
     assert view.breadcrumb.locations == BREADCRUMB_LOCATIONS["OverviewUtilization"]
     assert view.breadcrumb.is_displayed
+
+
+@pytest.mark.tier(1)
+@pytest.mark.provider([VMwareProvider], scope="module")
+@pytest.mark.parametrize("candu_db_restore", ["utilization_db.backup"], ids=["db"], indirect=True)
+def test_utilization_utilization_graphs(temp_appliance_extended_db, candu_db_restore, request,
+        entity):
+    """
+    Polarion:
+        assignee: gtalreja
+        casecomponent: Optimize
+        initialEstimate: 1/4h
+        testSteps:
+            1. Enable C&U
+            2. Wait until data will be collected
+            3. Go to Optimize/Utilization
+        expectedResults:
+            1.
+            2.
+            3. Verify that all graphs shows correctly
+    """
+    with temp_appliance_extended_db:
+        view = navigate_to(entity, "UtilTrendSummary")
+
+        assert view.summary.chart.is_displayed
+        assert view.details.cpu_chart.is_displayed
+        assert view.details.memory_chart.is_displayed
+        if entity == "regions":
+            assert view.details.disk_chart.is_displayed
+            assert view.report.disk_table.is_displayed
+        assert view.report.cpu_table.is_displayed
+        assert view.report.memory_table.is_displayed
```

Comment:
1)Please move this test to a new file in the tests/candu folder.
2)I would add a comment staing that this test automates the testing of the Overview->Utilization tab .
3)I would also rename the test to something like : test_utilization_trend()

Reply:
Thanks @nachandr , Done :)